contracts/netmap/contract.go

```diff
@@ -790,6 +802,18 @@ func UnusedCandidate() Candidate {
 	return Candidate{}
 }
 
+// GetEpochBlock returns block index when given epoch came. Returns -1 if the
+// epoch is missing.
+//
+// Use [LastEpochBlock] if you are interested in the current epoch.
+func GetEpochBlock(epoch int) int {
+	val := storage.Get(storage.GetReadOnlyContext(), append([]byte{epochIndexKey}, fourBytesBE(epoch)...))
+	if val == nil {
+		return -1
```

Comment:
Why not zero? It's never a valid block for an epoch as well. We use unsigned types for block numbers a lot and this can be confusing.

Reply:
zero seems better

how about declaring `uint32` return?

Reply:
`uint32` won't help much until we have NEP-25 properly implemented, so don't bother for now.

Reply:
switched to zero